components/user_settings/security/user_access_token_section/user_access_token_section.jsx

```diff
@@ -64,17 +65,19 @@ export default class UserAccessTokenSection extends React.Component {
         this.props.actions.getUserAccessTokensForUser(userId, 0, 200);
     }
 
-    UNSAFE_componentWillReceiveProps(nextProps) { // eslint-disable-line camelcase
-        if (!nextProps.active && this.props.active) {
-            this.setState({
+    static getDerivedStateFromProps(nextProps, prevState) {
+        if (!nextProps.active && prevState.active) {
```

Comment:
`prevState.active` starts falsey, so this block will never be ran. Can you change the `return null` below to always update the value of `active`?